LMGRemoteData/Classes/GraphQL/Queries/Fragments.graphql

```diff
@@ -163,6 +170,29 @@ fragment LocationItem on Location {
     }
 }
 
+fragment CollectionLocationItem on Location {
+    id
+    name
+    distance
+    address {
+      id
+      city
+      state
+      centre
+    }
+}
+
+fragment LocationsLocationItem on Location {
```

Comment:
This is just used in the map locations query right? Let's just call this `LocationItem` as the "main" one. Then all other fragments can have a pre-fix.